### DISCUSSION TITLE
FluentListBox - icon at the very end of the option

### DISCUSSION THREAD

**Discussion by Valkozaur** [microsoft/fluentui-blazor#2081] [Q&A] [answered]
Can an icon be set at the very end of the FluentListBox option:

```
<FluentListbox TOption="string" ValueChanged="(id) => _selectedGroup = _groups.FirstOrDefault(x => x.Id == Guid.Parse(id))">
    @foreach (var group in _groups)
    {
        <FluentOption Value="@group.Id.ToString()" >
            <FluentIcon Value="@(new Icons.Regular.Size16.Folder())" Slot="start" />
            @group.Name
            <FluentIcon Value="@GetIconByStatus(group.ProcessStatus)" Slot="end" />
        </FluentOption>
    }
</FluentListbox>
```

The result is: 
![image](https://github.com/microsoft/fluentui-blazor/assets/58659526/f98dcd25-dabb-481c-80f6-4991c7853ac9)

But I want the circle icon to be at the very end of the select option:
![image](https://github.com/microsoft/fluentui-blazor/assets/58659526/510c23d2-766e-4fa9-8d0e-aeedfe68a6d0)

**Comment by vnbaaij** [comment, marked as answer]
I think what could work is to add to your css:

```
fluent-option::part(content) {
   width: 100%;
}
```

Gives me this if I try in DevTools:
![image](https://github.com/microsoft/fluentui-blazor/assets/1761079/8a9d7c73-976c-4399-8e51-7a6ed84059d3)

**Reply by Valkozaur** [reply]
This worked like a charm, thank you a lot!
